operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -0,0 +1,185 @@
+# Create a Session Template to Boot Compute Nodes with Scalable Boot Projection Service (SBPS)
+
+When [compute nodes](../../glossary.md#compute-node-cn) are booted, the [Scalable Boot Projection Service (SBPS)](../../glossary.md#scalable-boot-projection-service-sbps)
+projects the root file system \(`rootfs`\) over the network to the compute nodes.ø
```

Comment:
There's a stray "ø" character at the end of this line.

Reply:
Removed